aws/table_aws_bedrock_foundation_model.go

```diff
@@ -0,0 +1,151 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/bedrock"
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+func tableAwsBedrockFoundationModel(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_bedrock_foundation_model",
+		Description: "AWS Bedrock Foundation Model",
+		List: &plugin.ListConfig{
+			Hydrate: listBedrockFoundationModels,
+			Tags:    map[string]string{"service": "bedrock", "action": "ListFoundationModels"},
+		},
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.SingleColumn("model_id"),
+			Hydrate:    getBedrockFoundationModel,
+			Tags:       map[string]string{"service": "bedrock", "action": "GetFoundationModel"},
```

Comment:
Would it make sense to handle the `ResourceNotFoundException` for this API call? According to the [documentation](https://docs.aws.amazon.com/bedrock/latest/APIReference/API_GetFoundationModel.html), this exception is thrown when the specified resource does not exist.